lib/Alchemy/Phrasea/Order/ValidationNotifier.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of Phraseanet
+ *
+ * (c) 2005-2014 Alchemy
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Alchemy\Phrasea\Order;
+
+use Alchemy\Phrasea\Model\Entities\Order;
+use Alchemy\Phrasea\Model\Entities\User;
+
+/**
+ * Interface ValidationNotifier
+ * @package Alchemy\Phrasea\Order
+ */
+interface ValidationNotifier
+{
+    /**
+     * @param Order $order
+     * @param User $recipient
+     */
```

Comment:
What about adding `@return void`? I don't like not having the `@return` docblock as you have to check whether it is intentionally omitted or not.